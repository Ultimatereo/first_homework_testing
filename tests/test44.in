5
4 0 2 0 4 0
3 5 5 1 4 1
2 5 3 1 2 4
0 1 5 2 5 4
2 2 1 4 5 3
