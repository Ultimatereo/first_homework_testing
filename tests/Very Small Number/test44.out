-49
36.3333
-52
-73.6666
75
